.infrastructure/helm-chart/todoapp/templates/pvc.yml

```diff
@@ -0,0 +1,13 @@
+
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{.Chart.Name}}-pvc
+  namespace: {{.Values.todoapp.namespace}}
+spec:
+  volumeName: {{.Chart.Name}}-pv
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: {{.Values.todoapp.pvcStorageRequest}}
```

Comment:
Ensure that `{{.Values.todoapp.pvcStorageRequest}}` is defined in your `values.yaml` file for the `todoapp`. If it's not defined, it could lead to persistent volume claim configuration issues.